src/shell/meson.build

```diff
@@ -51,3 +56,15 @@ server = executable(
     install_rpath: '$ORIGIN/../lib'
 )
 
+staging_args = ['./copy_staging.sh', '@BUILD_ROOT@', '@SOURCE_ROOT@', '@INPUT@']
+if have_cef
+    staging_args += cef_path
+endif
+
+staging = custom_target(
+    'staging',
+    input: server,
+    output: 'fake',
+    command: staging_args,
+    build_always: true,
```

Comment:
This is deprecated since Meson 0.47.0 (if you include `meson_version: '>= ...'` in your project() arguments it will ensure users have a new enough Meson, and will issue warnings if you use features newer than what you claim to support).

It's replaced by `build_always_stale: true` and `build_by_default: true`, which may not both be desired.